src/ORM/Table.php

```diff
@@ -1731,7 +1731,6 @@ public function updateAll(
             ->set($fields)
             ->where($conditions)
             ->execute();
-        $statement->closeCursor();
```

Comment:
I assume the RETURNING clause doesn't require this, but I don't know for sure.

Reply:
We can always add back the `closeCursor()` if we start having issues with inserts, which use `RETURNING` :slightly_smiling_face:.